src/main/java/org/terasology/launcher/util/DownloadUtils.java

```diff
@@ -204,8 +204,11 @@ public static boolean isJenkinsAvailable() {
                     throw new ConnectException();
```

Comment:
I don't get we throw an exception here - coudn't we just `return false;`?

Reply:
Well, I did it just to trigger its external `catch(IOException)` block, where logging happens. All other obstacles will lead to the same catch block too. Is there any problem with such flow?